src/styles/blocks/page.scss

```diff
@@ -0,0 +1,18 @@
+.page {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
+}
+
+.page__body {
+  margin: 0;
+}
+
+h1,
+h2 {
+  margin: 0;
+}
+
+span {
+  display: block;
+}
```

Comment:
Code Quality issue: Avoid using tag selectors. It's better to use classes instead. This is because tag selectors have a higher specificity and can lead to issues where your CSS doesn't apply as expected because there's a tag selector somewhere else that's overriding it.